CAPI/KubeadmConfig.yaml

```diff
@@ -24,3 +24,21 @@ spec:
           labels:
             +(cluster.x-k8s.io/watch-filter): "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-bootstrap-provider-kubeadm'].version | [0]}}"
             cluster.x-k8s.io/watch-filter: "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-bootstrap-provider-kubeadm'].version | [0]}}"
+  - name: kubeadmconfig-metrics
+    match:
+      resources:
+        kinds:
+        - bootstrap.cluster.x-k8s.io/v1alpha3/KubeadmConfig
+    context:
```

Comment:
Don't think you need any context, as it's not used

Reply:
Most likely no :D